scripts/qa_crowbarsetup.sh

```diff
@@ -2341,6 +2341,16 @@ function addfloatingip()
     nova add-floating-ip "$instanceid" "$floatingip"
 }
 
+# by setting --dns-nameserver for subnet, docker instance gets this as
+# DNS info (otherwise it would use /etc/resolv.conf from its host)
+function adapt_dns_for_docker()
+{
+    # DNS server is the first IP from the allocation pool, or the
+    # second one from the network range
+    local dns_server=`neutron subnet-show fixed | grep allocation_pools | cut -d '"' -f4`
```

Comment:
@MaximilianMeister No idea.  Your error message looks like more a Keystone error message than a neutron one.  For example, if want to show a missing subnet, I will read something like this instead:

    root@d52-54-01-77-77-01:~ # neutron subnet-show example
    Unable to find subnet with name 'example'

Is possible that your error message comes from a wrongly deployed neutron? [0]

[0] https://ask.openstack.org/en/question/46505/neutron-publicurl-endpoint-for-network-service-not-found/